core/src/main/java/org/mule/runtime/core/internal/event/BaseEventDecorator.java

```diff
@@ -181,18 +178,6 @@ public <T extends EventInternalContext> void setOperationPolicyContext(EventInte
     event.setOperationPolicyContext(context);
   }
 
-  /**
-   * Invoked after deserialization. This is called when the marker interface {@link DeserializationPostInitialisable} is used.
-   * This will get invoked after the object has been deserialized passing in the current MuleContext.
-   *
-   * @param muleContext the current muleContext instance
-   * @throws MuleException if there is an error initializing
-   */
-  @SuppressWarnings({"unused"})
-  private void initAfterDeserialisation(MuleContext muleContext) throws MuleException {
```

Comment:
Do we have a test asserting that we aren't breaking something with this?

Reply:
all existing tests that use a persistent object store